pinakes/data/email_template.html

```diff
@@ -0,0 +1,590 @@
+                              <td>$order_id</td>
+                            </tr>
+                            <tr>
+                              <th role="rowheader" scope="row">Order date:</th>
+                              <td>$order_date</td>
+                            </tr>
+                            <tr>
+                              <th role="rowheader" scope="row">Order time:</th>
+                              <td>$order_time</td>
+                            </tr>
+                            <tr>
+                              <th role="rowheader" scope="row">Product:</th>
+                              <td>$product</td>
+                            </tr>
+                            <tr>
+                              <th role="rowheader" scope="row">Portfolio:</th>
+                              <td>$portfolio</td>
+                            </tr>
+                            <tr>
+                              <th role="rowheader" scope="row">Platform:</th>
+                              <td>$platform</td>
+                            </tr>
+                          </tbody>
+                        </table>
+                      </td>
+                    </tr>
+                    <!-- end content block -->
+
+                    <!-- Content block -->
+                    <tr>
+                      <td class="rh-content__block">
+                        <strong><a href="$web_url/ui/catalog/approval/requests/detail/$approval_id" target="_blank">View more order details</a></strong> (requires authentication)
+                      </td>
+                    </tr>
+                    <!-- end content block -->
+
+                    <!-- Content block -->
+                    <tr>
+                      <td class="rh-content__block">
+                        <table align="center">
+                          <tr>
+                            <td class="rh-cta-link" align="center">
+                              <a target="_blank" href="$web_url/$approve_link">
+                                <span>
+                                  Process request
+                                </span>
+                              </a>
+                            </td>
+                          </tr>
+                        </table>
+                      </td>
+                    </tr>
+                    <!-- end content block -->
+
+                   <!-- end body section -->
+                  </table>
+                </td>
+              </tr>
+              <!-- end body section -->
+
+              <!-- Footer -->
+              <tr>
+                <td class="rh-footer">
+                  <table role="presentation" border="0" align="center" cellpadding="0" cellspacing="0" width="100%">
+                    <tr>
+                      <td class="rh-footer__text">
+                        This email was sent by Red Hat Ansible Automation Platform
```

Comment:
I'm not sure what language to use here, but we probably don't want to say "Red Hat Ansible Automation Platform". Maybe just "This email was sent by the Pinakes project"?